src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -370,6 +380,11 @@ Copyright (c) .NET Foundation. All rights reserved.
                                     Condition="'$(TieredCompilationQuickJitForLoops)' != ''"
                                     Value="$(TieredCompilationQuickJitForLoops)" />
 
+    <RuntimeHostConfigurationOption Include="System.Text.Encoding.EnableUnsafeUTF7Encoding"
+                                    Condition="'$(EnableUnsafeUTF7Encoding)' != ''"
+                                    Value="$(EnableUnsafeUTF7Encoding)"
+                                    Trim="true" />
```

Comment:
Just to confirm, "trim=true" means "this setting participates in the trimming process," not "enable trimming only if the value is set to _true_," correct?

Reply:
Correct. This value gets passed into the linker when `Trim=true` is set on it. See 

https://github.com/dotnet/sdk/blob/931137e5bf1d62863243c92b943cca38589c4bcc/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ILLink.targets#L172